wbooks/src/components/AppNavigator.tsx

```diff
@@ -0,0 +1,50 @@
+import 'react-native-gesture-handler';
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { RootStackParamList } from '@constants/types';
+import Colors from '@constants/colors';
+import Library from '@screens/Library';
+import Settings from '@screens/Settings';
+import BookDetail from '@screens/BookDetail';
+import { LIBRARY, SETTINGS } from '@constants/routes';
+import SCREEN_OPTS from '@app/constants';
+
+import TabIcon from './TabIcon';
```

Comment:
This is not correct, check the app's folder structure. The components or screens must be inside `components` or `screens` folder

`components/AppNavigator/components/TabIcon`